_spinnaker/terraform_integration.md

```diff
@@ -282,32 +288,30 @@ Github Enterprise:
 https://{host}/api/v3/repos/{org}/{repo}/contents/{file path}
 ```
 
-#### Terraform Versions
+#### Specifying the Terraform version in a stage
 
-Our terraform integration also supports selecting a version of terraform during a stage.  You can configure the version of terraform to run with the following example:
+This is optional and requires Armory Spinnaker 2.4.2 or later.. The Terraform Integration supports selecting a version of Terraform during a stage. You can configure the version of Terraform to run with the following example:
```

Comment:
..!

Reply:
I think it's actually required as of `2.4.2`. We stopped shipping a globally available version at some point. You have to supply a version now or things break.